controlpanel/frontend/views/dashboard.py

```diff
@@ -207,8 +207,19 @@ def form_invalid(self, form):
         return HttpResponseRedirect(self.get_success_url())
 
     def form_valid(self, form):
-        self.get_object().add_customers(form.cleaned_data["customer_email"])
+        not_notified = self.get_object().add_customers(
+            form.cleaned_data["customer_email"], self.request.user.justice_email
+        )
         messages.success(self.request, "Successfully added customers")
+
+        if len(not_notified) > 0:
+            messages.error(
+                self.request,
+                (
+                    f"Failed to notify {', '.join(not_notified)}. "
+                    "You may wish to email them your dahsboard link."
```

Comment:
```suggestion
                    "You may wish to email them your dashboard link."
```

Reply:
should we implement [CSpell](https://cspell.org/) via https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker?